beacon-chain/rpc/apimiddleware/custom_handlers.go

```diff
@@ -0,0 +1,159 @@
+package apimiddleware
+
+import (
+	"bytes"
+	"encoding/base64"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"strconv"
+	"strings"
+
+	"github.com/prysmaticlabs/prysm/shared/gateway"
+	"github.com/prysmaticlabs/prysm/shared/grpcutils"
+)
+
+type sszConfig struct {
+	sszPath      string
+	fileName     string
+	responseJson sszResponseJson
+}
+
+func handleGetBeaconStateSsz(m *gateway.ApiProxyMiddleware, endpoint gateway.Endpoint, writer http.ResponseWriter, request *http.Request) (handled bool) {
+	config := sszConfig{
+		sszPath:      "/eth/v1/debug/beacon/states/{state_id}/ssz",
```

Comment:
We should move these hardcoded strings as variables at the top of the file or somewhere else for more visibility and easier editing in the future, but no big deal in this PR